examples/crossplatform.pp

```diff
@@ -0,0 +1,89 @@
+case $facts['os']['family'] {
```

Comment:
Newer versions of Facter do support what they call "Modern Facts" as part of core:

https://docs.puppet.com/facter/3.5/core_facts.html#os

That said, I agree that using `facts[:osfamily]` would be better to retain backwards compatibility.